docs/content/modules/tensor_mechanics/systems/Kernels/StressDivergenceRSphericalTensors.md

```diff
@@ -1,8 +1,24 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#StressDivergenceRSphericalTensors
+#Stress Divergence RSpherical Tensors
+!description /Kernels/StressDivergenceRSphericalTensors
+
 !devel /Kernels/StressDivergenceRSphericalTensors float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
-!description /Kernels/StressDivergenceRSphericalTensors
+##Description
+This kernel solves the steady state stress divergence equation in polar coordinates on a 1D mesh.
+
+{!content/modules/tensor_mechanics/CommonDocumentation/StressDivergenceKernels.md!}
```

Comment:
You can do what you want, there is no getting around having a lot of content. We have a lot of code that needs to be documented, it is unavoidable. However, I would prefer if you would rename this directory to "common" or something in lower-case. Camel case items generally mirror the input syntax. This isn't a rule, put perhaps is should be just to help keep md files associated directly with C++ objects easier to identify. Just an idea.